lib/core/ng-package.json

```diff
@@ -7,7 +7,18 @@
     "./**/*.theme.scss"
   ],
   "whitelistedNonPeerDependencies": [
-    "cropperjs"
+    "cropperjs",
+    "@editorjs/editorjs",
```

Comment:
ADF core does not use "editorjs" in the code from what I see, why it should be added as non-peer dependency?